vars/pipelineNetworkApplyNonRestartChanges.groovy

```diff
@@ -0,0 +1,113 @@
+            lock(resource: params.DISABLE_LOCK ? "${Math.abs(new Random().nextInt(9999))}" : env.NET_NAME)
+            ansiColor('xterm')
+        }
+        environment {
+            PATH = "${env.WORKSPACE}/bin:${env.PATH}"
+        }
+        stages {
+            stage('CI Config') {
+                steps {
+                    sh "printenv"
+                    echo "params=${params.inspect()}"
+                    script {
+                        currentBuild.description = "action: ${params.ACTION}, node: ${params.NODE}"
+                        if (params.NODE?.toLowerCase() == 'all') {
+                            ANSIBLE_LIMIT = env.NET_NAME
+                        } else if (params.NODE?.trim()) {
+                            ANSIBLE_LIMIT = params.NODE.trim()
+                        } else {
+                            error "cannot run ansible: NODE parameter is not set"
+                        }
+                    }
+                }
+            }
+            stage('Checkout') {
+                parallel {
+                    stage('ansible'){
+                        steps {
+                            gitClone(
+                                vegaUrl: 'ansible',
+                                directory: 'ansible',
+                                branch: params.ANSIBLE_BRANCH,
+                            )
+                        }
+                    }
+                }
+            }
+            stage('Disable Alerts') {
+                steps {
+                    script {
+                        ALERT_SILENCE_ID = alert.disableAlerts(
+                            node: params.NODE,
+                            duration: 5, // minutes
+                        )
+                    }
+                }
+            }
+            stage('Apply Changes') {
+                environment {
+                    ANSIBLE_VAULT_PASSWORD_FILE = credentials('ansible-vault-password')
+                    HASHICORP_VAULT_ADDR = 'https://vault.ops.vega.xyz'
+                }
+                steps {
+                    withCredentials([usernamePassword(credentialsId: 'hashi-corp-vault-jenkins-approle', passwordVariable: 'HASHICORP_VAULT_SECRET_ID', usernameVariable:'HASHICORP_VAULT_ROLE_ID')]) {
+                        withCredentials([sshCredentials]) {
+                            sh label: "ansible playbooks/playbook-barenode-common.yaml", script: """#!/bin/bash -e
+                                ansible-playbook \
+                                    ${params.DRY_RUN ? '--check' : ''} \
+                                    --diff \
+                                    -u "\${PSSH_USER}" \
+                                    --private-key "\${PSSH_KEYFILE}" \
+                                    --inventory inventories \
+                                    --limit "${ANSIBLE_LIMIT}" \
+                                    --extra-vars '{"update_accounts": ${params.UPDATE_ACCOUNTS}}' \
```

Comment:
we may also want to add the following variable: `update_system_configuration` which seems to be unused here: https://github.com/vegaprotocol/ansible/pull/553/files#diff-8f5864cc92795bf24f967e88a1ef1189b25f33c870531b5bae24a6479d2fb612R33

Reply:
That is the issue in the other PR - I will remove `update_system_configuration` from there.